<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettingsDialog</class>
 <widget class="QDialog" name="ProjectSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project settings - Manuwrite</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="MainTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="MetaInfoTab">
      <attribute name="title">
       <string>Meta information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="MetaInformationLayout">
         <item>
          <widget class="QLabel" name="MetaInfoLabel">
           <property name="text">
            <string>Meta information</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="TitleLayout">
           <item>
            <widget class="QLabel" name="TitleLabel">
             <property name="text">
              <string>Title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TitleLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DateLayout">
           <item>
            <widget class="QLabel" name="DateLabel">
             <property name="text">
              <string>Date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="AuthorsLayout">
           <item>
            <widget class="QLabel" name="AuthorsLabel">
             <property name="text">
              <string>Authors</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="AuthorsLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="ProjectTypeLabel">
             <property name="text">
              <string>Project type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ProjectTypeValueLabel">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="DescriprionLabel">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="DescriptionPlainTextEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Additional meta information (YAML)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="AdditionalMetaInfoPlainTextEdit">
             <property name="placeholderText">
              <string>Put additional meta info here. It will be included in YAML metablock without any changes</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RenderTab">
      <attribute name="title">
       <string>Render</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="MetaInfoVerticalLayout">
         <item>
          <widget class="QLabel" name="RenderInfoLabel">
           <property name="text">
            <string>Render settings</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="ChooseFilesHorizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="FilesToRenderVerticalLayout">
             <item>
              <widget class="QLabel" name="FilesToRenderLabel">
               <property name="text">
                <string>Files to render</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="FilesToRenderListWidget"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="ChooseFilesButtonsLayout">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="MoveAllUpToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="MoveUpToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="MoveLeftToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="MoveRightToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="MoveDownToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="MoveAllDownToolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="AllFilesVerticalLayout">
             <item>
              <widget class="QLabel" name="AllFilesLabel">
               <property name="text">
                <string>All Markdown files in project</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="AllFilesListWidget"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="StyleHorizontalLayout">
           <item>
            <widget class="QLabel" name="StyleLabel">
             <property name="text">
              <string>Style</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="StyleCombobox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="RenderToHorizontalLayout">
           <item>
            <widget class="QLabel" name="RenderToLabel">
             <property name="text">
              <string>Render to</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="RenderToCombobox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="PandocCommandHorizontalLayout">
           <item>
            <widget class="QLabel" name="PandocCommandLabel">
             <property name="text">
              <string>Pandoc command</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="GeneratePandocCommandButton">
             <property name="text">
              <string>Generate pandoc command</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="PandocCommandLineEdit">
             <property name="toolTip">
              <string>Auto generated part of the command</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="PandocCommandManualLineEdit">
             <property name="toolTip">
              <string>Manual part of the command</string>
             </property>
             <property name="placeholderText">
              <string>Add manual arguments here...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionMoveToTheTop">
   <property name="text">
    <string>Move to the top</string>
   </property>
  </action>
  <action name="actionMoveUp">
   <property name="text">
    <string>Move up</string>
   </property>
  </action>
  <action name="actionMoveLeft">
   <property name="text">
    <string>Move left</string>
   </property>
  </action>
  <action name="actionMoveRight">
   <property name="text">
    <string>Move right</string>
   </property>
  </action>
  <action name="actionMoveDown">
   <property name="text">
    <string>Move down</string>
   </property>
  </action>
  <action name="actionMoveToTheBottom">
   <property name="text">
    <string>Move to the bottom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
